File: ChristmasJoy.App/src/app/models/Comment.ts
import { UserStatus } from './UserStatus';

export class Comment {
    id: string;
    fromUserId: number;
    content: string;
    toUserId: number;
    commentDate: Date;
    likes: number[]; // array of user ids
    fromUserStatus: UserStatus;
    isAnonymous = false;
    isPrivate = false;
}

File: ChristmasJoy.App/Models/SqLiteModels/Comment.cs
using System;
using System.Collections.Generic;

namespace ChristmasJoy.App.Models.SqLiteModels
{
    public class Comment
    {
        public int Id { get; set; }
    
        public int FromUserId { get; set; }
      
        public string Content { get; set; }
    
        public CommentType CommentType { get; set; }
    
        public int ToUserId { get; set; }
    
        public DateTime CommentDate { get; set; }
    
        public List<Like> Likes { get; set; }

        public bool IsAnonymous { get; set; }
  }
}
